kalite/distributed/baktests/__init__.py

```diff
@@ -0,0 +1,5 @@
+from ajax_messages_test import *
```

Comment:
Ugh, some files seem to have snuck in. Will have to do a clean up.

On Wed, 20 May 2015 17:33 Michael Gallaspy <notifications@github.com> wrote:

> In kalite/distributed/baktests/__init__.py
> <https://github.com/learningequality/ka-lite/pull/3649#discussion_r30763788>
> :
>
> > @@ -0,0 +1,5 @@
> > +from ajax_messages_test import *
>
> What are baktests?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/learningequality/ka-lite/pull/3649/files#r30763788>.
>